FindDict/FindDict/Sources/Scences/Game/GameVC.swift

```diff
@@ -190,13 +201,28 @@ final class GameVC: ViewController {
         self.present(guessedRightWordVC, animated: true)
     }
     
-    private func cropImage(origin: CGPoint, size: CGSize){
-        //        print("origin",origin)
-        //        print("size",size)
-        let cropRect = CGRect(origin: origin, size: size)
-        guard let imageRef = image.image?.cgImage?.cropping(to: cropRect) else { return  };
-        let newImage = UIImage(cgImage: imageRef, scale: image.image!.scale, orientation: image.image!.imageOrientation)
-        cropImageView.image = newImage
+    
+    func resizeImage(image: UIImage, size: CGSize, x: CGFloat, y: CGFloat) -> CGImage {
+        UIGraphicsBeginImageContext(size)
+        image.draw(in:CGRect(x: x, y: y, width: size.width, height:size.height))
+        let renderImage = UIGraphicsGetImageFromCurrentImageContext()
+        UIGraphicsEndImageContext()
+        
+        guard let resultImage = renderImage?.cgImage else {
+            print("image resizing error")
+            return UIImage().cgImage!
+        }
+        return resultImage
+    }
+    
+    private func cropImage(_ title: String, _ cropRect: CGRect){
+        let newImage = resizeImage(image: image.image!, size: buttonLayer.bounds.size, x: buttonLayer.bounds.origin.x, y: buttonLayer.bounds.origin.y)
+        guard let imageRef = newImage.cropping(to: cropRect) else { return  };
+        let newUIImage = UIImage(cgImage: imageRef, scale: 1, orientation: self.image.image!.imageOrientation)
```

Comment:
```suggestion
        let resizedImage = resizeImage(image: image.image!, size: buttonLayer.bounds.size, x: buttonLayer.bounds.origin.x, y: buttonLayer.bounds.origin.y)
        guard let croppedImageRef = resizedImage.cropping(to: cropRect) else { return  };
        let croppedUIImage = UIImage(cgImage: croppedImageRef, scale: 1, orientation: self.image.image!.imageOrientation)
```
처럼 어떤 작업이 수행되어 만들어진 값인지 의미를 담은 변수명으로 수정해주시면 코드를 이해하기 더 편해질 것 같습니다!

Reply:
wow 친절히 수정까지.......